src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -39,17 +40,22 @@ internal TagHelperExecutionContext(string tagName)
         /// <param name="executeChildContentAsync">A delegate used to execute the child content asynchronously.</param>
         /// <param name="startWritingScope">A delegate used to start a writing scope in a Razor page.</param>
         /// <param name="endWritingScope">A delegate used to end a writing scope in a Razor page.</param>
+        /// <param name="selfClosing">
+        /// The <c>bool</c> indicating whether or not the tag in the Razor soruce is self-closing.
```

Comment:
- don't use `<c>` to refer to a type: `<see cref="bool"/>` instead
- leave out "The"
- nit: "is" -> "was"